apps/extension/src/App/AppContent.tsx

```diff
@@ -57,7 +57,7 @@ export const AppContent = ({ warnings }: Props): JSX.Element => {
   }, [accounts, passwordInitialized, accountLoadingStatus]);
 
   return (
-    <Stack className="py-5 max-w-84" full gap={6}>
+    <Stack className="py-5" full gap={6}>
```

Comment:
One thing: If we remove this, the Width becomes unconstrained on several Firefox views. Maybe we should leave this until that can be solved without this override